package.json

```diff
@@ -202,7 +202,6 @@
     "@babel/eslint-plugin": "^7.17.7",
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@babel/preset-env": "^7.22.4",
-    "@esbuild/linux-x64": "^0.21.5",
```

Comment:
we dont need it?

Reply:
`npm install` failed for me on mac because of this. I asked @smallst and we decided that I'll remove and he'll handle the linux issue on his side.